app/components/place_hold_checkbox_wrapper_component.rb

```diff
@@ -1,15 +1,15 @@
 # frozen_string_literal: true
 
 class PlaceHoldCheckboxWrapperComponent < ActionView::Component::Base
-  def initialize(holdables: nil)
-    @holdables = holdables
+  def initialize(volumetric_calls: nil)
+    @volumetric_calls = volumetric_calls
```

Comment:
👌 liked volumetric_class better than holdables, nice!

Reply:
It's definitely more accurate and uses the terminology of the domain rather than inventing a new term.